nuttx-configs/aerofc-v1/nsh/defconfig

```diff
@@ -1408,10 +1409,12 @@ CONFIG_NSH_DISABLE_PRINTF=y
 # CONFIG_NSH_DISABLE_PSSTACKUSAGE is not set
 # CONFIG_NSH_DISABLE_PUT is not set
 # CONFIG_NSH_DISABLE_PWD is not set
+# CONFIG_NSH_DISABLE_REBOOT is not set
```

Comment:
Yes it would but, it does not consider PX4's usage.  for now y is the right answer on this one. 
The px4 reboot_main will be used. The nsh cmd_reboot will then be stripped.